uaa/src/main/webapp/WEB-INF/spring/saml-providers.xml

```diff
@@ -301,6 +301,10 @@
 
     <bean id="fixedHttpMetaDataProvider" class="org.cloudfoundry.identity.uaa.provider.saml.FixedHttpMetaDataProvider" />
 
+    <bean id="defaultSamlConfig" class="org.cloudfoundry.identity.uaa.provider.saml.SamlConfigurationBean">
+        <property name="signatureAlgorithm" value="${login.saml.signatureAlgorithm:SHA1}"/>
```

Comment:
Is this to be SHA256?

Reply:
Same thought here :) I guess the state was SHA-1 before but would be good to have a plan to adapt a secure default.

Reply:
Should the defualt value be SHA256. the default being SHA1 is not secure. This is not terrible since it is overridden, but the default should be closed to SHAKE256 or SHA3-512

Reply:
In production SHA1 was never supposed to be used. In OpsManager, you can set the value to SHA256 or SHA512 only. SHA1 was exposed only because we removed the bean.

Reply:
@torsten-sap Is this a PR you can approve as acceptable?